sonar-db/src/main/java/org/sonar/db/ce/CeTaskInputDao.java

```diff
@@ -58,7 +59,7 @@ public void insert(DbSession dbSession, String taskUuid, InputStream data) {
     ResultSet rs = null;
     DataStream result = null;
     try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this "try" to a try-with-resources. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2093)